.styleguidist/styleguidist.sections.js

```diff
@@ -146,6 +146,12 @@ module.exports = {
                 'SingleSelect/SingleSelect',
               ]),
             },
+            {
+              name: 'LoadingSpinner',
```

Comment:
What should the official name for the component be? Was it decided on somewhere? In the designs it's Preloader.